memlab.bib

```diff
@@ -110,6 +110,98 @@ @string{vn
+	Title = {Some principles of memory schemata},
+	Year = {1975}}
+
+@article{Mann19,
+	Author = {Manning, J.R.},
+	Date-Added = {2019-02-06 12:17:21 -0500},
+	Date-Modified = {2019-02-06 12:18:00 -0500},
+	Journal = {PsyArXiv},
+	Title = {Episodic memory: mental time travel or a quantummemory wave'function?},
+	Year = {2019}}
+
+@article{StraEtal14,
+	Author = {Strange, B.A. and Witter, M.P. and Lein, E.S and Moser, E.I.},
+	Date-Added = {2019-02-05 23:45:08 -0500},
+	Date-Modified = {2019-02-05 23:46:59 -0500},
+	Journal = {Nature Reviews Neuroscience},
+	Number = {10},
+	Pages = {655},
+	Title = {Functional organization of the hippocampal longitudinal axis},
+	Volume = {15},
+	Year = {2014}}
+
+@article{RichEtal14,
+	Author = {Richards, B.A. and Xia, F. and Santoro, A. and Husse, J. and Woodin, M.A. and Josselyn, S.A. and Frankland, P.W.},
+	Date-Added = {2019-02-05 22:05:36 -0500},
+	Date-Modified = {2019-02-05 22:07:17 -0500},
+	Journal = {Nature Neuroscience},
+	Number = {7},
+	Pages = {981},
+	Title = {Patterns across multiple memories are identified over time},
+	Volume = {17},
+	Year = {2014}}
+
+@article{TompDava17,
+	Author = {Tompary, A. and Davachi, L.},
+	Date-Added = {2019-02-05 21:08:39 -0500},
+	Date-Modified = {2019-02-05 22:52:40 -0500},
+	Journal = {Neuron},
+	Number = {1},
+	Pages = {228-241},
+	Title = {Consolidation promotes the emergence of representational overlap in the hippocampus and medial prefrontal cortex},
+	Volume = {96},
+	Year = {2017}}
+
+@article{HassEtal15,
+	Author = {Hasson, U. and Chen, J. and Honey, C. J.},
+	Date-Added = {2019-02-04 22:54:05 -0500},
+	Date-Modified = {2019-02-04 22:56:06 -0500},
+	Journal = {Trends in Cognitive Science},
+	Number = {6},
+	Pages = {304-315},
+	Title = {Hierarchical process memory: memory as an integral component of information processing},
+	Volume = {19},
+	Year = {2015}}
+
+@article{KimEtal15,
+	Author = {Kim, S. and Ogawa, K. and Lv, J. and Schweighofer, N. and Imamizu, H.},
+	Date-Added = {2019-02-04 19:28:32 -0500},
+	Date-Modified = {2019-02-04 19:30:50 -0500},
+	Journal = {PLoS biology},
+	Number = {12},
+	Pages = {e1002312},
+	Title = {Neural substrates related to motor memory with multiple timescales in sensorimotor adaptation},
+	Volume = {13},
+	Year = {2015}}
+
+@article{BromEtal10c,
+	Author = {Bromberg-Martin, E. and Matsumoto, M. and Nakahara, H. and Hikosaka, O.},
+	Date-Added = {2019-02-04 19:12:57 -0500},
+	Date-Modified = {2019-02-04 19:15:57 -0500},
+	Journal = {Neuron},
+	Number = {3},
+	Pages = {499-500},
+	Title = {Multiple timescales of memory in lateral habenula and dopamine neurons},
+	Volume = {67},
+	Year = {2010}}
+
+@inproceedings{Kipp01,
+	Author = {Kipp, M.},
+	Booktitle = {Seventh European Conference on Speech Communication and Technology},
+	Date-Added = {2019-02-03 15:09:28 -0500},
+	Date-Modified = {2019-02-03 15:24:47 -0500},
+	Title = {Anvil-a generic annotation tool for multimodal dialogue},
+	Year = {2001}}
+
 @article{Shap19,
```

Comment:
There's already a Shap19 key-- change the existing key to Shap19a and this one to Shap19b

Reply:
Shap19 was not added or modified in this pull request